### DISCUSSION TITLE
AttributeError: 'ProgressBar' object has no attribute 'main_progress_bar' when use neuralprophet().fit(), can u help me? thanks

### DISCUSSION THREAD

**Discussion by stanhaishan** [ourownstory/neural_prophet#1562] [Q&A - get help using NeuralProphet]
my code:
m = NeuralProphet(
    trend_global_local="local",
    season_global_local="local",
    changepoints_range=0.8,
    trend_reg=5,
    n_forecasts=3,
    n_lags=10,
    unknown_data_normalization=True,
)
m.set_plotting_backend("matplotlib")

df_train, df_test = m.split_df(dff, valid_p=0.33, local_split=True) # dataframe: ['ID', 'ds', 'y']
metrics = m.fit(df_train, freq='D', validation_df=df_test, learning_rate=0.001, progress='plot', metrics=True, early_stopping=True)

error:
![image](https://github.com/ourownstory/neural_prophet/assets/28746452/0c7bcebf-0c65-41fa-9d77-c8881323fe1b)

**Comment by Delmark04** [comment]
I have encountered the same error, what helped me is to restart kernel. I assume, that I have made some other training processes in the same environment and some global variable was changed, first of all I tried reinstalling NeuralProphet, but it didn't help. And then I figured out that I just need to restart kernal
